chat/src/main/java/net/md_5/bungee/chat/ComponentSerializer.java

```diff
@@ -158,11 +159,27 @@ public static JsonElement toJson(ComponentStyle style)
         return gson.toJsonTree( style );
     }
 
+    /**
+     * @param object the object to serialize
+     * @return the JSON string representation of the object
+     * @deprecated Error-prone, be careful which object you input here
+     */
+    @Deprecated
     public static String toString(Object object)
     {
         return gson.toJson( object );
     }
 
+    public static String toString(Content content)
```

Comment:
Would plugins ever need this? Both seem kind of internal, can they be hidden away, package-private maybe?

Reply:
Can't make this package-private, its accessed from `HoverEvent` which is in `api.chat` while the serializer is only in `chat`.
Yes, plugins won't need it.